citation_template.html.rsp

```diff
@@ -1,59 +1,102 @@
-	    <header>
-	    	<h1 class="display-3 p-2">
-	    		<a href=https://parkingreform.org style="background-color: #ff4747; color: #ffffff; text-transform: uppercase;text-decoration-line:none">Parking Reform Network</a>
-	    	</h1>
-	    </header>
-	    <div class="container site-container-wrap">
-			<h1><%=city%>, <%=state%></h1>
-		    <% for(j in 1:nrow(city_df)) { %>
-		        <h2> Citation <%= j %></h2>
-		        <h3>Summary</h3>
-		        <p><%= city_df$Summary[j] %></p>
-		        <h3>Source Description </h3>
-		        <p><%= city_df$Source.Description[j] %></p>
-		        <h3>Type</h3>
-		        <p><%= city_df$Type[j] %></p>
-		        <h3>URL</h3>
-		        <p><%= city_df$URL[j] %></p>
-		        
-		        <% if(nchar(city_df$Attachments[j]) > 0) { %>
-		            <h3>Attachments</h3>
-		            <% attachments <- strsplit(city_df$Attachments[j], "\\s+") [[1]] %>
-		            <% for(k in 1:length(attachments)) { %>
-		                <% file_type <- str_extract(attachments[k], "\\.[a-zA-Z]+$") %>
-		                <% img_path <- paste("attachment_images/", city_state_no_space, "_", j, "_", k, file_type, sep = "") %>
-		                <% download.file(attachments[k], paste("city_detail/", img_path, sep =""))  %>
-		                <img src = "<%= img_path %>" class="attachments img-fluid">
-		            <% } %>
-		        <% } %>
-		    <% } %>
+                    <dt class="col-12 col-sm-4 col-lg-3">Type</dt>
+                    <dd class="col-12 col-sm-8 col-lg-9"><%= city_df$Type[j] %></dd>
 
-	    </div>
+                    <dt class="col-12 col-sm-4 col-lg-3">URL</dt>
+                    <dd class="col-12 col-sm-8 col-lg-9"><%= city_df$URL[j] %></dd>
+                </dl>
+                <% if(nchar(city_df$Attachments[j]) > 0) { %>
+                    <h3>Attachments</h3>
+                    <ul class="mt-4 list-unstyled">
+                    <% attachments <- strsplit(city_df$Attachments[j], "\\s+") [[1]] %>
+                    <% for(k in 1:length(attachments)) { %>
+                        <li class="attachment">
+                            <% file_type <- str_extract(attachments[k], "\\.[a-zA-Z]+$") %>
+                            <% img_path <- paste("attachment_images/", city_state_no_space, "_", j, "_", k, file_type, sep = "") %>
+                            <% download.file(attachments[k], paste("city_detail/", img_path, sep =""))  %>
+                            <img class="img-fluid" src="<%= img_path %>">
+                        </li>
+                    <% } %>
+                    </ul>
+                <% } %>
+            </section>
+        <% } %>
+        </div>
     </div>
 
+    <!-- Bootstrap Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous"></script>
```

Comment:
moved JS to end of file per recommendation: https://getbootstrap.com/docs/5.1/getting-started/introduction/#js

Reply:
you rule